zenhappens/src/images/burger-menu-close.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think I did that but how would I be able to check?

Reply:
If you see any stuff about your generator and/or Sketch, that usually means it can still be optimized further, e.g.:
```
    <!-- Generator: Sketch 50 (54983) - http://www.bohemiancoding.com/sketch -->
    <title>Group</title>
    <desc>Created with Sketch.</desc>
```

Reply:
ok! realized the 2 that aren't optimized, I don't use them. So I deleted them now :D